File: Exercício_19.c
#include <stdio.h>
#include <stdlib.h>

int main(int argc, char *argv[])
{
float ac, ve, t , V;

printf("\nDigite a velocidade inicial: ");
scanf("%f", &ve);

printf("\nDigite a aceleracao: ");
scanf ("%f", &ac);

printf("\nDigite o tempo do percurso: ");
scanf("%f", &t);

V = ve + (ac * t);

  if ( V > 120)
  {
  printf("\nVelocidade muito alta");
  }

  else
  {
  if ((V <= 120 ) && ( V > 80))
  {
  printf("\nVelocidade alta");
  }
      
        else if (( V > 60) && (V <= 80)){
        printf("\nVelocidade de cruzeiro");
        }
else
{
        if((V <= 60) && (V > 40))
        {
        printf("\nVelocidade permitida");
        }
        else
        {
        printf("\nVelocidade muito baixa");
        }
}
}
  
  return 0;
}

File: Exercício_19.py
a=float(input ('digite a aceleração          '))
t=float(input ('digite o tempo de percurso   '))
V0=float(input ('digite a velocidade inicial  '))

V=(V0+(a*t))

if(V>120):
    print(f" veículo muito rápido ")

elif((V<=120) and (V>80)):
    print(f"veículo rápido ")

elif((V>60) and (V<=80)):
    print("velocidade de cruzeiro")

elif((V<=60) and (V>40)):
    print("velocidade permitida")

else:
    print("veículo muito lento")